src/main/java/crazypants/enderio/conduit/AbstractConduitNetwork.java

```diff
@@ -203,15 +214,16 @@ public void sendBlockUpdatesForEntireNetwork() {
 
           if (!sidesToNotify.isEmpty()) {
             tracker.start("ForgeEventFactory.onNeighborNotify() at " + pos);
-            boolean canceled = ForgeEventFactory.onNeighborNotify(world, pos, bs, sidesToNotify).isCanceled();
+            // TODO Set the 4th parameter to only update the redstone state when the conduit network has a redstone conduit network in it
+            boolean canceled = ForgeEventFactory.onNeighborNotify(world, pos, bs, sidesToNotify, false).isCanceled();
             tracker.stop();
 
             if (!canceled) {
               for (EnumFacing side : WEDUNS) {
                 if (sidesToNotify.contains(side)) {
                   final BlockPos offset = pos.offset(side);
                   tracker.start("World.notifyNeighborsOfStateChange() from " + pos + " to " + offset + " (" + world.getBlockState(offset) + ")");
-                  world.notifyBlockOfStateChange(offset, blockType);
+                  world.notifyNeighborsOfStateChange(offset, blockType, false);
```

Comment:
wrong one! you want neighborChanged(BlockPos pos, final Block p_190524_2_, BlockPos p_190524_3_)

Reply:
Oh okay, I had that in an earlier version locally but changed it because of the tracker text. That makes sense